src/tools/tests/golden_kt.arcs

```diff
@@ -0,0 +1,18 @@
+// Example manifest file, serving as a test for the schema2kotlin and schema2cpp
+// code generators.
+meta
+  namespace: arcs.golden
+
+particle Gold
```

Comment:
Might be nice to rename this to something like `KotlinPrimitivesGolden`, just to make it clear how it's different from the other one (when looking at the generated code)